debezium-core/src/main/java/io/debezium/data/geometry/Geography.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.data.geometry;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+
+/**
+ * A semantic type for a Geography class.
+ * Used to describe geometries operating on a spherical base (rather than Geometry which operates on a planar basis).
+ * This is extremely similar to a Geometry – but they're quite different types and not in a hierarchy.
+ *
+ * WKB has to be forwards-compatible to the spec, so strict OGC WKB, and EWKB (PostGIS) are both ok.
+ *
+ * @author Robert Coup
+ */
+public class Geography {
+    public static final String LOGICAL_NAME = "io.debezium.data.geometry.Geography";
+
+    public static final String WKB_FIELD = "wkb";
+    public static final String SRID_FIELD = "srid";
+
+    /**
+     * Returns a {@link SchemaBuilder} for a Geography field. You can use the resulting SchemaBuilder
+     * to set additional schema settings such as required/optional, default value, and documentation.
+     *
+     * @return the schema builder
+     */
+    public static SchemaBuilder builder() {
+        return SchemaBuilder.struct()
+                            .name(LOGICAL_NAME)
+                            .version(1)
+                            .doc("A Geography")
+                            .field(WKB_FIELD, Schema.BYTES_SCHEMA)
+                            .field(SRID_FIELD, Schema.INT32_SCHEMA);
+    }
+
+    /**
+     * Returns a {@link SchemaBuilder} for a Geography field, with all other default Schema settings.
+     *
+     * @return the schema
+     * @see #builder()
+     */
+    public static Schema schema() {
+        return builder().build();
+    }
+
+    /**
+     * Create a value for this schema using WKB and SRID
+     * @param geomSchema a {@link Schema} instance which represents a Geography; may not be null
+     * @param wkb OGC Well-Known Binary representation of the geometry; may not be null
```

Comment:
What's OGC?

Reply:
The [Open Geospatial Consortium](http://www.opengeospatial.org/) — worldwide standards body for geospatial.

Reply:
Well-Known-Binary is defined in the [Simple Features for SQL](https://en.wikipedia.org/wiki/Simple_Features) standard